### PR TITLE
FEATURE: QA-18253 Cover some bugs found

### PR DESCRIPTION
**Pull request by TienBien**
JIRA: QA-18253

---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13344/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15016/ : FAILURE in 7m 45s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13345/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15017/ : FAILURE in 7m 40s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13351/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15023/ : FAILURE in 7m 35s

**Comment by hungnguyenauto** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13353/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15026/ : FAILURE in 8m 12s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13355/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15028/ : FAILURE in 7m 33s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13356/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15029/ : FAILURE in 7m 34s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13360/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15033/ : FAILURE in 7m 32s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13362/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15034/ : SUCCESS in 7m 40s

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardTiger.spec.ts [hunk 1]
```diff
@@ -2,50 +2,107 @@
...
+                widget.waitTableLoaded().click();
+                widgetConfig.openConfiguration().checkFilterCheckbox(STAGE_NAME_CHECKBOX);
+                widget.isLoading(true);
+                widgetConfig.checkFilterCheckbox(STAGE_NAME_CHECKBOX);
+                widget.isLoading(true);
+            },
+        );
+
+        //Cover ticket: RAIL-4772
+        it(
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardTiger.spec.ts:92]: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboard.spec.ts
this test is duplicated with test in above link. Why don't you put them in a same place, use { tags: ["checklist_integrated_tiger", "checklist_integrated_bear"] },

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drilling.spec.ts [hunk 2]
```diff
@@ -129,3 +132,49 @@ describe("Drilling", { tags: ["post-merge_integrated_bear"] }, () => {
...
+            editMode.edit();
+            widget.waitChartLoaded().click();
+            widgetConfig.openInteractions().addInteraction("Sum of Probability", "measure");
+            widgetConfig
+                .getDrillConfigItem("Sum of Probability")
+                .chooseAction("Drill into URL")
+                .openCustomUrlEditor();
+            new CustomURLDialog().hasItem("Created");
+
+            //Write test with table insight here.
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drilling.spec.ts:168]: What is it?

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drilling.spec.ts [hunk 3]
```diff
@@ -129,3 +132,49 @@ describe("Drilling", { tags: ["post-merge_integrated_bear"] }, () => {
...
+        Navigation.visitCopyOf("dashboard/drill-to-insight");
+        editMode.edit();
+        widget.waitChartLoaded().click();
+        widgetConfig.openInteractions().getDrillConfigItem("Sum of Velocity").remove();
+        editMode.save(true).edit();
+        widget.waitChartLoaded().click();
+        widgetConfig.openInteractions().hasInteractionItems(false);
+    });
+
+    it(
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drilling.spec.ts:155]: which ticket this test cover?

File: libs/sdk-ui-tests-e2e/cypress/tools/attributeFilterConfig.ts [hunk 4]
```diff
@@ -73,4 +80,14 @@ export class AttributeFilterConfiguration {
...
         return new Button(".gd-attribute-filter-apply-button__next.s-save");
     }
+
+    cancel() {
+        cy.get(".dropdown-body .cancel-button").click();
+        return this;
+    }
+
+    save() {
+        cy.get(".s-attribute-filter-dropdown-configuration-save-button").click();
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/attributeFilterConfig.ts:90]: why don't you use `getSaveButton().click()`

File: libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts [hunk 5]
```diff
@@ -157,6 +176,16 @@ export class TopBar {
         this.clickShareButton();
         return this;
     }
+
+    deleteDashboard(confirm: boolean) {
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts:180]: should be written in dashboardHeader.ts

File: libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts [hunk 6]
```diff
@@ -147,6 +156,16 @@ export class TopBar {
         return this;
     }
 
+    isEditing(expect: boolean): this {
+        cy.get(".s-gd-dashboard-title.is-editing").should(expect ? "exist" : "not.exist");
+        return this;
+    }
+
+    hasDashboardTitlePlaceholder(text: string): this {
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts:164]: i don't see it is used anywhere

File: libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts [hunk 7]
```diff
@@ -147,6 +156,16 @@ export class TopBar {
         return this;
     }
 
+    isEditing(expect: boolean): this {
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts:159]: i don't see it is used anywhere

File: libs/sdk-ui-tests-e2e/cypress/tools/widget.ts [hunk 8]
```diff
@@ -120,6 +125,11 @@ export class Widget {
         return this;
     }
 
+    click() {
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/widget.ts:128]: I think this should be renamed to focus

File: libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts [hunk 9]
```diff
@@ -184,6 +191,11 @@ export class WidgetConfiguration {
         this.getElement().find(".gd-input-field").clear().type(text);
         return this;
     }
+
+    checkFilterCheckbox(text: string) {
```
Comment (NghiLamGD) [inline libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts:195]: param should be name as attributeFilter, or item

File: libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts (hunk 5 above)
Comment (TienBien) [reply libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts:180]: I think it belongs in TopBar() in dashboards.ts is reasonable.

File: libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts (hunk 9 above)
Comment (TienBien) [reply libs/sdk-ui-tests-e2e/cypress/tools/widgetConfiguration.ts:195]: I have rename it to _selectFilterCheckbox()_ . It more clearly about the action and the object that the function performs.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13407/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15075/ : FAILURE in 8m 07s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13411/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15078/ : FAILURE in 7m 31s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13433/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15092/ : FAILURE in 7m 45s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13463/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15125/ : FAILURE in 7m 37s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13474/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15135/ : FAILURE in 8m 23s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13494/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15154/ : FAILURE in 7m 29s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13495/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15155/ : FAILURE in 8m 08s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13496/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15156/ : SUCCESS in 7m 37s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13497/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15157/ : SUCCESS in 7m 48s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13519/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15179/ : FAILURE in 7m 33s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13520/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15180/ : FAILURE in 7m 28s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13521/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15181/ : FAILURE in 7m 45s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13530/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15190/ : SUCCESS in 8m 48s

File: libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts (hunk 5 above)
Comment (NghiLamGD) [reply libs/sdk-ui-tests-e2e/cypress/tools/dashboards.ts:170]: dashboardHeader.ts and TopBar are duplicated. Should move this method to dashboardHeader
then create a follow-up ticket to move all functions in TopBar to dashboardHeader.ts

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13573/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15225/ : FAILURE in 7m 41s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13574/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15226/ : FAILURE in 7m 49s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13576/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15228/ : FAILURE in 7m 45s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13579/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15232/ : SUCCESS in 7m 43s

**Comment by NghiLamGD** [comment]
LGTM

**Review by maitrieuphilong — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3737

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2728/ : SUCCESS in 12m 54s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15263/ : SUCCESS in 8m 04s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/263/ : UNSTABLE in 11m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3885/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/610/ : FAILURE in 1m 24s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/233/ : UNSTABLE in 12m 52s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/119/ : SUCCESS in 5m 11s (non-voting)

**Comment by TienBien** [comment]
extended test - tiger-cypress - record attributeFilterConfig.spec.ts,dashboard.spec.ts,dashboardTiger.spec.ts,drilling.spec.ts

**Comment by yenkins** [comment]
**Cypress recording succeeded** :tada:

Here is your new recording: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-record-zuul/73/artifact/libs/sdk-ui-tests-e2e/recordings/mappings/TIGER/*zip*/archive.zip

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardTiger.spec.ts**|11(4 skipped)|-|82.350s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-record-zuul/73/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13619/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/759/ : SUCCESS in 6m 29s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13651/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15302/ : FAILURE in 7m 47s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13653/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15304/ : SUCCESS in 7m 48s

**Comment by TienBien** [comment]
extended test - cypress - isolated dashboardTiger.spec.ts

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13658/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15307/ : SUCCESS in 7m 44s

**Comment by TienBien** [comment]
extended test - cypress - isolated dashboardTiger.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13659/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/766/ : UNSTABLE in 12m 09s

**Comment by TienBien** [comment]
extended test - tiger-cypress - isolated dashboardTiger.spec.ts

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13660/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/767/ : SUCCESS in 6m 39s

**Review by maitrieuphilong — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3737

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2740/ : SUCCESS in 18m 00s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15308/ : SUCCESS in 7m 46s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/278/ : UNSTABLE in 11m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3897/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/622/ : FAILURE in 2m 02s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/247/ : SUCCESS in 8m 43s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/128/ : SUCCESS in 4m 34s (non-voting)

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13668/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15315/ : FAILURE in 7m 43s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13669/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15316/ : FAILURE in 7m 38s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13670/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15317/ : SUCCESS in 7m 39s

**Comment by TienBien** [comment]
extended test - tiger-cypress - isolated dashboard.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13671/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/771/ : UNSTABLE in 28m 33s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13693/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15335/ : SUCCESS in 7m 42s

**Comment by TienBien** [comment]
extended test - cypress - isolated dashboard.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13696/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/779/ : UNSTABLE in 25m 25s

**Comment by TienBien** [comment]
extended test - cypress - record dashboard.spec.ts

**Comment by TienBien** [comment]
extended test - cypress - record dashboard.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13699/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/781/ : FAILURE in 8m 33s

**Comment by TienBien** [comment]
extended test - cypress - record dashboard.spec.ts

**Comment by yenkins** [comment]
**Cypress recording succeeded** :tada:

Here is your new recording: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-record-zuul/55/artifact/libs/sdk-ui-tests-e2e/recordings/mappings/BEAR/*zip*/archive.zip

**Comment by TienBien** [comment]
extended test - cypress - isolated dashboard.spec.ts

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13703/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/783/ : SUCCESS in 6m 49s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13702/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15340/ : SUCCESS in 8m 04s

**Comment by TienBien** [comment]
extended test - cypress - isolated attributeFilterConfig.spec.ts,dashboardKPI.spec.ts,drilling.spec.ts

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13704/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/784/ : UNSTABLE in 15m 06s

**Comment by TienBien** [comment]
extended test - cypress - record attributeFilterConfig.spec.ts,dashboardKPI.spec.ts,drilling.spec.ts

**Comment by yenkins** [comment]
**Cypress recording succeeded** :tada:

Here is your new recording: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-record-zuul/56/artifact/libs/sdk-ui-tests-e2e/recordings/mappings/BEAR/*zip*/archive.zip

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardKPI.spec.ts**|9(1 skipped)|-|132.555s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-record-zuul/56/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13705/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/785/ : SUCCESS in 7m 58s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by TienBien** [comment]
extended test - cypress - isolated attributeFilterConfig.spec.ts,dashboardKPI.spec.ts,drilling.spec.ts,dashboard.spec.ts

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13707/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15342/ : FAILURE in 7m 39s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13708/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/786/ : SUCCESS in 8m 55s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/13709/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15343/ : SUCCESS in 7m 44s

**Review by maitrieuphilong — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline gate aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by maitrieuphilong — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3737

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2745/ : SUCCESS in 8m 47s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15347/ : SUCCESS in 7m 40s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/292/ : SUCCESS in 11m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3902/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/627/ : FAILURE in 1m 45s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/253/ : SUCCESS in 8m 00s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/133/ : SUCCESS in 5m 16s (non-voting)

**Pull request merged**